cpp/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ include(cmake/Modules/ConfigureCUDA.cmake) # set other CUDA compilation flags
 # * dependencies ----------------------------------------------------------------------------------
 
 rapids_find_package(
-  legate REQUIRED Legion LegionRuntime
+  legate
```

Comment:
```suggestion
  legate REQUIRED
```

Reply:
Isn't Legate still required?

Reply:
I'll revert this.  I am honestly not sure whether it is, but this was an (unsuccessful) try to fix the linking issue with current legate.